src/udiskslinuxdriveata.c

```diff
@@ -434,7 +434,7 @@ selftest_status_to_string (SkSmartSelfTestExecutionStatus status)
   return ret;
 }
 
-static gboolean get_pm_state (UDisksLinuxDevice *device, GError **error, guchar *count)
+gboolean get_pm_state (UDisksLinuxDevice *device, GError **error, guchar *count)
```

Comment:
Please add doc strings - every public method should have its purpose, input arguments and result values documented. E.g. it should be noted what the result `count` values mean and what values are possible.